arangod/Replication/ReplicationMetricsFeature.cpp

```diff
@@ -33,94 +33,96 @@ namespace arangodb {
+          "arangodb_replication_tailing_markers", 0,
+          "Number of replication tailing markers processed")),
       _numTailingProcessedDocuments(
-        server.getFeature<arangodb::MetricsFeature>().counter(
-          "arangodb_replication_tailing_documents", 0, "Number of replication tailing document inserts/replaces processed")),
-      _numTailingProcessedRemovals(
-        server.getFeature<arangodb::MetricsFeature>().counter(
-          "arangodb_replication_tailing_removals", 0, "Number of replication tailing document removals processed")),
-      _numTailingBytesReceived(
-        server.getFeature<arangodb::MetricsFeature>().counter(
-          "arangodb_replication_tailing_bytes_received", 0, "Number of bytes received for replication tailing requests")),
-      _numFailedConnects(
-        server.getFeature<arangodb::MetricsFeature>().counter(
-          "arangodb_replication_failed_connects", 0, "Number of failed connection attempts and response errors during replication")),
-      _waitedForTailing(
-        server.getFeature<arangodb::MetricsFeature>().counter(
+          server.getFeature<arangodb::MetricsFeature>().counter(
+              "arangodb_replication_tailing_documents", 0,
+              "Number of replication tailing document inserts/replaces "
+              "processed")),
+      _numTailingProcessedRemovals(server.getFeature<arangodb::MetricsFeature>().counter(
+          "arangodb_replication_tailing_removals", 0,
+          "Number of replication tailing document removals processed")),
+      _numTailingBytesReceived(server.getFeature<arangodb::MetricsFeature>().counter(
+          "arangodb_replication_tailing_bytes_received", 0,
+          "Number of bytes received for replication tailing requests")),
+      _numFailedConnects(server.getFeature<arangodb::MetricsFeature>().counter("arangodb_replication_failed_connects",
+                                                                               0,
+                                                                               "Number of failed connection attempts and response errors during replication")),
+      _waitedForTailing(server.getFeature<arangodb::MetricsFeature>().counter(
           "arangodb_replication_tailing_request_time", 0,
           "Wait time for replication tailing requests [ms]")),
-      _waitedForTailingApply(
-        server.getFeature<arangodb::MetricsFeature>().counter(
+      _waitedForTailingApply(server.getFeature<arangodb::MetricsFeature>().counter(
           "arangodb_replication_tailing_apply_time", 0,
-          "Time needed to apply replication tailing data [ms]")) {
+          "Time needed to apply replication tailing data [ms]")),
+      _syncTimeTotal(server.getFeature<arangodb::MetricsFeature>().counter(
+          "arangodb_replication_synchronous_requests_total_time", 0,
+          "Total time needed for all synchronous replication requests "
+          "accumulated.")),
```

Comment:
Let's add the unit, so it is more understandable for the end user.
```suggestion
          "accumulated [ns]")),
```